src/layer/vector/SVG.js

```diff
@@ -71,13 +65,6 @@ export var SVG = Renderer.extend({
 		delete this._svgSize;
 	},
 
-	_onZoomStart: function () {
-		// Drag-then-pinch interactions might mess up the center and zoom.
```

Comment:
I'm worried about removing this, just because I still have nightmares about https://github.com/Leaflet/Leaflet/issues/3798.

I don't have a touchscreen with me to test this. Can somebody else confirm this PR doesn't break this kind of touch interactions for the SVG renderer?